content/posts/2022-03-18-opi-event.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Videos and Slides From OPI Event"
+date: 2022-03-17T15:06:30-06:00
+---
+
+The Open Programmable Infrastructure Alpha Event was a great success thanks to
+all of our presentors. We had great participation. See below for all of the
+videos and slides, which are hosted on the [IPDK YouTube page](https://www.youtube.com/channel/UCrabFf05cRQgbmmk7aLzlJQ).
+
+# Day One
+
+## OPI Event Keynote: Nick McKeown
+
+{{< youtube qi-BaWXfrqg >}}
```

Comment:
does it work ? `{{< youtube qi-BaWXfrqg >}}`

Reply:
Yes! It's the same format from the [old Diamond-bluff repo.](https://github.com/Diamond-Bluff/diamond-bluff.github.io/blob/main/content/posts/2022-03-18-opi-event.md), which you can see live [here](https://diamond-bluff.github.io/posts/2022-03-18-opi-event/).